src/preferences/dialog/dlgprefbroadcastdlg.ui

```diff
@@ -392,6 +392,20 @@
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
+           <item row="7" column="0" colspan="2">
+            <widget class="QCheckBox" name="enableFileBroadcast">
+             <property name="text">
+              <string>Broadcast metadata in file</string>
```

Comment:
"Write NowPlaying.txt file"
The tooltips should give the full path.
Or schod the path and fikename also optional?

Reply:
Well it's a toggle right? So it is optional. I thought about the line edit so users can copy and paste the path to the broadcasting app where necessary.